WalletWasabi.Tests/Helpers/ServiceFactory.cs

```diff
@@ -67,7 +67,14 @@ public static KeyManager CreateKeyManager(string password = "blahblahblah")
 		KeyPath segwitAccountKeyPath = KeyManager.GetAccountKeyPath(Network.Main, ScriptPubKeyType.Segwit);
 		ExtPubKey segwitExtPubKey = extKey.Derive(segwitAccountKeyPath).Neuter();
 
-		return new KeyManager(encryptedSecret, extKey.ChainCode, masterFingerprint, segwitExtPubKey, null, skipSynchronization: true, 21, blockchainState, null, segwitAccountKeyPath, null);
+		ExtPubKey? taprootExtPubKey = null;
+		if (isTaprootAllowed)
+		{
+			KeyPath taprootAccountKeyPath = KeyManager.GetAccountKeyPath(Network.Main, ScriptPubKeyType.TaprootBIP86);
+			taprootExtPubKey = extKey.Derive(taprootAccountKeyPath).Neuter();
+		}
```

Comment:
Sadly this is because the tests have hardcoded values that assume Wasabi is segwit only. I didn't update the tests because I wanted to prove that everything was still working after the big refactoring but we must fix the test to make them work with taproot because that's how Wasabi really works.

Reply:
Yes, I agree. This is a debt we will pay after the release. Tests will fail in the transaction building section - you know that part the best.